### PR TITLE
Font

### PR DESCRIPTION
**Pull request by heartjigi**
expend default font weight

upper or equal 700 -> 'SFUIText-Bold'
upper or equal 600 -> 'SFUIText-Semibold'
upper or equal 400 -> 'SFUIText-Regular'
upper or equal 200 -> 'SFUIText-Light'
upper or equal 100 -> 'SFUIText-Ultrathin'
other ->'SFUIText-Regular'

And fix error on api util
handle state 200 and null response case

### CODE REVIEW

File: src/components/Login/Login.js [hunk 1]
```diff
@@ -189,6 +190,8 @@ class Login extends Component {
       } else {
         this.setState({ loaded: true });
       }
+    } else {
+      this.setState({ loaded: true });
```
Comment (ifndefdeadmau5) [inline src/components/Login/Login.js:194]: When does this else statement used?

File: src/components/SignUp/GeneralInfo.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
   },
   nextTxt: {
-    fontFamily: 'SFUIText-Bold',
+  	fontWeight: 'bold',
```
Comment (ifndefdeadmau5) [inline src/components/SignUp/GeneralInfo.js]: Seems need re-formatting

File: src/components/Login/Login.js (hunk 1 above)
Comment (heartjigi) [reply src/components/Login/Login.js:194]: when server give 200 and no json...
Actually  it is some error state, but we should handle any unexpected state.

File: src/components/SignUp/GeneralInfo.js (hunk 2 above)
Comment (heartjigi) [reply src/components/SignUp/GeneralInfo.js]: it fit just in this ///

**Comment by MechanicKim** [comment]
I think this convention is okay.
if (condition) 1 line statement;
or
if (condition)
  1 line statement;

But style guide and convention say that we should use '{', '}'

I reviewed it.
How about you @SarahhhC @heartjigi ?

File: src/components/Login/Login.js (hunk 1 above)
Comment (ifndefdeadmau5) [reply src/components/Login/Login.js:194]: Thank you for explanation

**Comment by MechanicKim** [comment]
Good work! LGTM

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/98/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

**Comment by ifndefdeadmau5** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/98/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

**Pull request merged**